### PR TITLE
Update account documentation

### PR DESCRIPTION
**Pull request by mikestone14**

### CODE REVIEW

File: source/includes/_accounts.md [hunk 1]
```diff
@@ -39,18 +41,27 @@ This endpoint creates an account
 
 ### Parameters
 
+Parameter | Required | Default | Description
+--------- | -------  | ------- | ----------
+number | true | N/A | The account number
```
Comment (BlakeWilliams) [inline source/includes/_accounts.md]: Thoughts on formatting this like a table? I think there's a vim plugin that helps

File: source/includes/_accounts.md [hunk 2]
```diff
@@ -120,3 +132,74 @@ A successful GET will return an HTTP 200
 <aside class="warning">
 An unsuccessful GET will return an HTTP 404
 </aside>
+
+## Update a specific account
+
+This endpoint updates an individual account
```
Comment (BlakeWilliams) [inline source/includes/_accounts.md]: Should we specify that giving specific interval or summary usages overwrites?

Comment (mikestone14) [reply source/includes/_accounts.md]: Yeah that's a good idea

File: source/includes/_accounts.md [hunk 3]
```diff
@@ -39,18 +41,27 @@ This endpoint creates an account
...
-location | true | The address lines 1 & 2 for the account
-state | true | The account geographic state's 2 digit abbreviation
-load_zone | true | The account's load zone
-utility | true | The account's utility
-load_profile | true | The account's load profile
-voltage | true | The account's voltage
-rate_class | true | The account's rate class
-capacity_tag_kw | false | The account's capacity tag in kWh
-transmission_tag_kw | false | The account's transmission tag in kWh
+| Parameter                | Required | Default | Description                                         |
```
Comment (mikestone14) [inline source/includes/_accounts.md]: @BlakeWilliams look how beautiful these tables are now 💅

File: source/includes/_accounts.md [hunk 4]
```diff
@@ -120,3 +132,78 @@ A successful GET will return an HTTP 200
 <aside class="warning">
 An unsuccessful GET will return an HTTP 404
 </aside>
+
+## Update a specific account
+
+This endpoint updates an individual account
+
+<aside class="warning">
+Updating an account with new usage data erases all existing usage data
```
Comment (mikestone14) [inline source/includes/_accounts.md]: @BlakeWilliams do you think this is a sufficient warning?

File: source/includes/_accounts.md (hunk 3 above)
Comment (BlakeWilliams) [reply source/includes/_accounts.md]: 😍

File: source/includes/_accounts.md (hunk 4 above)
Comment (BlakeWilliams) [reply source/includes/_accounts.md]: Sounds good 👍

**Pull request merged**